docs/package.json

```
@@ -9,12 +9,20 @@
     "help": "docusaurus --help",
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "write-i18n": "npm run write-heading-ids && npm run write-translations",
     "format": "prettier --check \"**/*.{js,jsx,json,md,scss}\"",
     "format:fix": "prettier --write \"**/*.{js,jsx,json,md,scss}\"",
     "lint": "set -ex; eslint .",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "crowdin": "crowdin",
+    "crowdin:download": "npm run write-i18n && npm run crowdin download && npm run format:fix",
+    "crowdin:sync": "npm run write-i18n && npm run crowdin upload && npm run crowdin download && npm run format:fix",
```

I don't think we need to format files that aren't getting checked into git, and is there a need to write translations before downloading the latest?
```suggestion
    "crowdin:download": "crowdin download",
    "crowdin:upload": "npm run write-i18n && crowdin upload",
```

---

Linting broke the build in Travis when the files were not formatted correctly (while they were checked in).
Don't know if Travis will ignore them now when `i18n` is in `.gitignore`.

---

Added your changes, let's try it out in the next master build % ).